modules/openy_features/openy_prgf/modules/openy_prgf_3c/openy_prgf_3c.info.yml

```diff
@@ -3,6 +3,7 @@ description: 'Implements a paragraph with 3 columns.'
 type: module
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
 dependencies:
```

Comment:
This paragraph has module `openy_block_branch_amenities` in the list of dependencies
Do we really need it here? I think make sense to investigate why this dependency is listed here.

Reply:
Fixed